arangod/RestHandler/RestReplicationHandler.cpp

```diff
@@ -701,6 +702,12 @@ void RestReplicationHandler::handleCommandClusterInventory() {
   std::vector<std::shared_ptr<LogicalCollection>> cols = ci.getCollections(dbName);
   VPackBuilder resultBuilder;
   resultBuilder.openObject();
+
+  DatabaseFeature& databaseFeature = _vocbase.server().getFeature<DatabaseFeature>();
+  TRI_vocbase_t* vocbase = databaseFeature.lookupDatabase(dbName);
```

Comment:
In theory `vocbase` can be a nullptr here.
At least `lookupDatabase` can return `nullptr`. We should add a check here.